docs/more-pushmsg.md

```diff
@@ -115,3 +115,101 @@ axios({
     }
   });
 ```
+
+### Java
+
+With maven dependency:
+```xml
+<dependency>
+    <groupId>com.fasterxml.jackson.core</groupId>
+    <artifactId>jackson-databind</artifactId>
+    <version>2.9.8</version>
+</dependency>
+```
+And code:
+```java
+package com.gotify.client;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.BufferedReader;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class GotifyClient {
+
+    // example main method
+    public static void main(String[] args) throws IOException {
+        GotifyClient client = new GotifyClient("http://localhost:8008/message?token=<apptoken>");
+        client.sendMessage("Hi there", "Hello from Java", 5);
+    }
+
+    private String gotifyUrl;
+
+    public GotifyClient(String gotifyUrl) {
+        this.gotifyUrl = gotifyUrl;
+    }
+
+    public void sendMessage(String msg, String title, int priority) throws IOException {
+        Message message = new Message(msg, priority, title);
+        ObjectMapper mapper = new ObjectMapper();
+        doHTTPPost(gotifyUrl, mapper.writeValueAsString(message));
+    }
+
+    private static String doHTTPPost(String urlString, String params) throws IOException {
+        URL url = new URL(urlString);
+        HttpURLConnection yc = (HttpURLConnection) url.openConnection();
+        yc.setRequestMethod("POST");
+        yc.setRequestProperty("Content-Type", "application/json");
+        yc.setDoOutput(true);
+        DataOutputStream wr = new DataOutputStream(yc.getOutputStream());
+        wr.writeBytes(params);
```

Comment:
The ObjectMapper actually can write directly to the stream, so we could skip the conversion to string.